lms/lmsdb/models.py

```diff
@@ -143,6 +153,46 @@ def random_password(cls, stronger: bool = False) -> str:
     def get_notifications(self) -> Iterable['Notification']:
         return Notification.fetch(self)
 
+    def select_notes_of_user(self) -> List[Dict[str, Any]]:
```

Comment:
External user still can't understand how `User.notes` is different from `User.select_notes_of_user`.

Reply:
In order to check if a manager or a staff tries to delete a private note of another manager or staff